financialaid/serializers.py

```diff
@@ -119,37 +122,52 @@ class FinancialAidActionSerializer(serializers.Serializer):
         write_only=True
     )
     tier_program_id = IntegerField(write_only=True)
+    justification = ChoiceField(
+        choices=FinancialAidJustification.ALL_JUSTIFICATIONS,
+        default=None,
+        write_only=True
+    )
 
     def validate(self, data):
         """
         Validators for this serializer
         """
-        # Check that the previous financial aid status allows for the new status
-        if (data['action'] == FinancialAidStatus.APPROVED and
-                self.instance.status != FinancialAidStatus.PENDING_MANUAL_APPROVAL):
-            raise ValidationError("Cannot approve application that is not pending manual approval.")
-        if (data['action'] == FinancialAidStatus.PENDING_MANUAL_APPROVAL and
-                self.instance.status not in [FinancialAidStatus.PENDING_DOCS, FinancialAidStatus.DOCS_SENT]):
-            raise ValidationError("Cannot mark documents as received for an application awaiting docs.")
-        # Check tier program exists
-        try:
-            data["tier_program"] = TierProgram.objects.get(
-                id=data["tier_program_id"],
-                program_id=self.instance.tier_program.program_id,
-                current=True
-            )
-        except TierProgram.DoesNotExist:
-            raise ValidationError("Financial Aid Tier does not exist for this program.")
+        # Required field
+        if data.get("action") is None:
+            raise ValidationError({"action": "This field is required."})
```

Comment:
Can you add tests to cover this line and the two other `This field is required` validation errors below?